test/all-tests.coffee

```diff
@@ -60,7 +56,3 @@ require './task-helpers.spec'
 require './dom-helpers.spec'
 require './helpers/string.spec'
 require './helpers/period.spec'
-
-# The whole Boom Boom.
-# This should be done **last** because it starts up the whole app
-require './router.spec'
```

Comment:
I removed the router spec.  It loads the entire application, which includes the less files.  Since it wasn't testing very much I felt the tradeoff on not adding the less loader to karma was worth it.